ts/features/bonus/siciliaVola/screens/voucherGeneration/VoucherGeneratedScreen.tsx

```diff
@@ -91,6 +92,19 @@ const VoucherGeneratedScreen = (props: Props): React.ReactElement | null => {
   switch (voucherGenerated.kind) {
     // TODO: manage conflict and other state when the final swagger is available
     case "success":
+      const backButtonProps = {
+        primary: false,
+        bordered: true,
+        onPress: props.completed,
+        title: I18n.t("global.buttons.exit")
+      };
+
+      const continueButtonProps = {
+        primary: true,
+        onPress: () => props.stampaVoucher(voucherGenerated.value.id),
+        title: I18n.t("global.genericSave"),
+        blocked: isLoading(pdfVoucherState)
```

Comment:
Did you test it?
If you use the type declaration
`const continueButtonProps: BlockButtonProps`
you'll get an error because blocked doesn't exist as a valid property
Maybe you mean `disabled`?

Reply:
Sorry, my fault!🙏🏼
I only tested, and modified, the other path but I forgot this!

Fixed in [71de1ee](https://github.com/pagopa/io-app/pull/3439/commits/71de1ee6b4d760578017af62d72eefe24b52fe02)